tracer/src/Datadog.Trace.Tools.Runner/CheckIisCommand.cs

```diff
@@ -123,6 +127,44 @@ internal static async Task<int> ExecuteAsync(CheckIisSettings settings, string a
                     Utils.WriteWarning(IisMixedRuntimes);
                 }
 
+                if (process.Modules.Any(m => Path.GetFileName(m).Equals("aspnetcorev2_outofprocess.dll", StringComparison.OrdinalIgnoreCase)))
+                {
+                    // IIS site is hosting aspnetcore in out-of-process mode
+                    // Trying to locate the actual applicative process
```

Comment:
application process?

```suggestion
                    // Trying to locate the actual application process
```